src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps;
+
+import com.google.sps.classes.Utility;
```

Comment:
Just curious, do we need this line if we already have the package `com.google.sps`? I don't really know if it can work that way, but I saw the tests walkthrough didn't import the specific file, and that they didn't need to write the name of the class before each function call, probably because of the omission of this line.

Reply:
I tried doing a test without importing the involved classes and I got the "cannot find symbol" error. I also noticed the tests in the walkthrough but I don't know what the difference is.